### ISSUE TITLE
Problem: Dspace transfers failing at verify transfer compliance in qa/1.x

### ISSUE THREAD

**Issue by ross-spencer** [archivematica/Issues#1071]
**Expected behaviour**

DSpace transfers work in `qa/1.x` the way they do in `stable/1.10`. 

**Current behaviour**

They are currently failing unexpectedly during `verify transfer compliance` where the same transfer will work in `stable/1.11`.

The failing microservice is: `Check if file or folder` (`Module fileToFolder_v1.0`)

With the error output:

```
Cannot move file /var/archivematica/sharedDirectory/currentlyProcessing/ITEM@2429-1521.zip to folder /var/archivematica/sharedDirectory/currentlyProcessing/ITEM@2429-1521 because it already exists
```

![image](https://user-images.githubusercontent.com/1880412/72672750-1fd1bd80-3a2d-11ea-92c0-218fd683c6e5.png)

**Steps to reproduce**

1. From the sample-data select `ITEM@2429-1521.zip` from the `SampleTransfers/DSpace` folder and configure it to run as a DSpace transfer.
2. It will fail almost immediately with this output. 

**Your environment (version of Archivematica, operating system, other relevant details)**

1. Error condition tested in Docker-compose, and AM111Xenial running up to date `qa/1.x` code.
2. Working condition tested in sandbox@archivematica.org running `1.10.1`. 

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] A testing plan for this issue has been implemented and passed (testing plan information should be included in the issue body or comments)
- [ ] Documentation regarding this issue has been written and merged (if applicable)
- [ ] Details about this issue have been added to the release notes (if applicable)

**Comment by sallain** [comment]
Alas, this is still happening. Tested on qa on bionic.

**Comment by replaceafill** [comment]
This seems to be caused by [this change](https://github.com/artefactual/archivematica/blob/46273947514507d7c51bba581fe0ff6e1966af4d/src/MCPServer/lib/server/packages.py#L298-L316) introduced in https://github.com/artefactual/archivematica/pull/1446 where the processing configuration is preserved in compressed files. The extraction directory for a compressed transfer (like a Dspace one) is created in advance at this point.

But this clashes with the `fileToFolder_v1.0` client script (which is the starting point of Dspace transfers) that [expects the target directory to not exist](https://github.com/artefactual/archivematica/blob/46273947514507d7c51bba581fe0ff6e1966af4d/src/MCPClient/lib/clientScripts/file_to_folder.py#L35-L44).

As far as I can tell `fileToFolder_v1.0` is only used in the Dspace workflow, so I think it's reasonable to update it and remove this check.

**Comment by sallain** [comment]
Oh interesting. I just want to rephrase this to ensure that I understand:

The purpose of `fileToFolder_v1.0` is: "Given a transfer that is one file, move the file into a directory with the same basename and updated the DB for the new path."

When you start a DSpace transfer you can select either a directory of material, or an individual ZIP file (I'm working on an update to the docs that explains this better - see [here](https://github.com/artefactual/archivematica-docs/pull/351). `fileToFolder_v1.0` is expecting the latter case, an individual ZIP. The problem in the original comment above *doesn't* occur if you attempt the former (a directory of material), because `fileToFolder_v1.0` doesn't need to run. I've just tested this to confirm.

In the past, the script had to create the extraction directory at the time that the transfer started. The change in artefactual/archivematica#1446 means that the extraction directory is already created when the script runs, so the script is like "what the hell, there's already a directory here!" And then it errors out and everyone cries.

To sum up, removing the script should fix the problem, since all the script does is create the extraction directory, which is already being created.

Therefore, I think, go ahead and remove the action and we'll cross our fingers that it works!

**Comment by sallain** [comment]
DSpace transfers now start! Tested on qa/0.x on Bionic.
